### PR TITLE
ci: validate ruby gems

### PR DESCRIPTION
**Pull request by jon-shipley**
Include ruby gem test

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,25 +1,51 @@
...
+dist: bionic
 os: linux
 language: node_js
 services:
-- docker
+  - docker
 node_js:
-- 12.18.2
-env:
-  jobs:
```
Comment (GuyHarwood) [inline .travis.yml:9]: this env matrix enables parallel jobs, are you intentionally disabling that?

Comment (jon-shipley) [reply .travis.yml:9]: Yes, it is disabled as we need to run 2 languages: node and ruby.  I'm probably missing something; the Travis documentation is pretty bad at showing useful examples.

**Comment by activemq** [comment]
LGTM 👍

**Review by activemq — APPROVED** [review]

**Review by GuyHarwood — APPROVED** [review]
LGTM

**Pull request merged**